operator/main.py

```diff
@@ -899,6 +920,12 @@ def deploy_csi_pods(core_v1_client):
 
     if api_instance.major > "1" or api_instance.major == "1" and \
        api_instance.minor >= "22":
+
+        if csi_driver_object_api_version() is not None and \
+           csi_driver_object_api_version() != "v1":
+            filename = os.path.join(MANIFESTS_DIR, "csi-driver-object.yaml")
```

Comment:
- we know the csi driver name is `kadalu` no need to use file
- just perform `k delete csidriver kadalu`
- current condition will fail if we are updating from <=1.14

Pls don't mind, being extra cautious due to gluster mount options debacle 😅

Reply:
Sure ack.

Reply:
lib_execute(*cmd) works or do you recommend to use non file use based commands on utis_execute()?

Reply:
fine with anything that works and there's no need for a function for that single command 😅